gr-qtgui/lib/timeRasterGlobalData.cc

```diff
@@ -158,9 +159,9 @@ double TimeRasterData::value(double x, double y) const
 
     double _y = floor(top - y);
     double _loc = _y * (d_cols) + x + d_resid;
-    auto location = static_cast<ssize_t>(_loc);
+    auto location = static_cast<std::ptrdiff_t>(_loc);
```

Comment:
Ugh, `ptrdiff`. It's signed, but it's not actually a pointer diff we're using it for.

How about just:

```c++
    const auto location = static_cast<size_t>(_loc);
    if (location < d_data.size()) {
        return d_data[location];
    }
    return 0.0;
``` 

(getting rid of the needless `returnValue` variable, since multiple `return` statements is not in fact an antipattern)

Reply:
Great! I'll take whatever you say because I was just guessing here.

Reply:
Confirming that this change has successfully built on the conda Windows CI PR.